tools/importer/cmsis_importer.json

```diff
@@ -149,7 +149,8 @@
       "a019acaf8d6fb1f0512414d072f667cc2749b1d9",
       "a884fdc0639ae4e17299838ec9de4fddd83cf93c",
       "6c827cb5879bc096e45efd992dfadcb96c1d50bc",
-      "919282322e106b82fea50878f41b6c75a7eb356b"
+      "919282322e106b82fea50878f41b6c75a7eb356b",
```

Comment:
Is there a way forward without addingn extra commits here?

Reply:
@bulislaw - Can work on that if you have any suggestions. In this case the code is picked from sample implementation and wont even compile as is. 
https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/RTOS2/RTX/Examples/TrustZoneV8M/RTOS/CM33_s/tz_context.c#L29

I have 3 changes listed below of which I have solution for two, but still for header file we will need a commit, unless you want to track it some other way.

1. Update Stack size to 512: Can add PR to CMSIS repo
2. Config options : Can remove completely (Cannot add to config files which reside inside RTOS, as the entire folder is skipped during secure lib build)
3. Add correct header file.

Reply:
Could you do this: https://github.com/ARMmbed/mbed-os/pull/6273/commits/f072380b35b287ffb3a0effcdaa85ccac1571103

Reply:
You probably won't need the `_RTE_` for current version of CMSIS

Reply:
@bulislaw - https://github.com/ARMmbed/mbed-os/pull/6483/commits/b93aca294646a72fe8d91f8801a41c288b362a3b Will have to add this change in json file.

Reply:
Why? Use the `#include "RTE_Components.h"` and `#include CMSIS_device_header` the same way my commit does and configure it in config file. Is there something i'm missing?

Reply:
@bulislaw - Sorry I missed the header part, but the change I wanted to address was conditional compilation. https://github.com/ARMmbed/mbed-os/commit/b93aca294646a72fe8d91f8801a41c288b362a3b#diff-ae477765d86108e79533ea6617f50737R28

File should be compiled only in case of secure build and not for non-secure build, will check with @theotherjimmy if we can have some option from tools

Reply:
Yep. We can have that support in the tools. We currently don't have something like that. Would you like that as part of this PR? or another PR?